src/code/qualification/generateOutputFolders.m

```diff
@@ -0,0 +1,66 @@
+function  [SectionsOut, Id] = generateOutputFolders(SectionsIn, REOutputFolder)
+%GENERATEOUTPUTFOLDERS Support function: 
+% Explore tree of sections and generate folders of sections within REOutputFolder 
+% linearize the section structure
+%
+%   generateOutputFolders(SectionsIn, REOutputFolder)
+%       SectionsIn (structure or cell) sections to be created
+%       REOutputFolder (string) name of folder where the sections
+%
+
+% Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org
+
+%% ---------------------------------------------------% 
+
+% Check for Sections Class, 
+% if cell: multiple sections, if struct, only one available
+if iscell(SectionsIn)    
+
+% Loop on every section
+for i = 1:length(SectionsIn)
+
+    % Get Section Id
+    Id(i) = SectionsIn{i}.Id;
+    SectionsOut(Id(i)).Id=Id(i);

```

Comment:
Is it really supposed to be `SectionsOut(Id(i))` and not `SectionsOut(i)` ? 
E.g. call the function with the sections-json below: SectionsOut will return 6 entries, 3 of them invalid 
(because section ids are defined as 4,5,6). 

```
{
  "Sections": [
    {
      "Id": 5,
      "Title": "Chapter 1",
      "Content": "../Content/Section1.md"
    },
    {
      "Id": 4,
      "Title": "Chapter 2",
      "Content": "../Content/Section2.md",
      "Sections": [
        {
          "Id": 6,
          "Title": "Chapter 2.1",
          "Content": "../Content/Section2.1.md"
        }
      ]
    }
  ]
}
```